readme.md

```diff
@@ -54,20 +54,24 @@ Yields:
 
 ```js
 { '1':
-   [ { type: 'insert',
+   [
```

Comment:
If this section is going to be reformatted, it may make sense to use https://prettier.io/ to align with what other projects in `syntax-tree` and `unified` do for styling.

Reply:
yes ok i don't formatted the other lines, because it gets too verbose, it was my intention to add better visibility to the "a key can be an array fact"

it takes me hours before i find out it was not a bug that a loop over keys not print some inserts before i see that was an surprising sub array

Reply:
updated with prettier.io formatting

Reply:
Thanks!